forage-android/src/main/java/com/joinforage/forage/android/LDManager.kt

```diff
@@ -85,4 +95,26 @@ internal object LDManager {
         internalLogger.i("[LaunchDarkly] Vault type set to $vaultType")
         return vaultType as VaultType
     }
+
+    internal fun getPollingIntervalsFeatureFlag(): IntArray {
```

Comment:
nit: Can you remove the "FeatureFlag" portion of this function name so it's the same style as `getVaultProvider`?